reg/drone/phy/acoustics/Note.0.1.uavcan

```diff
@@ -1,19 +1,9 @@
 # Description of a generic musical note in terms of basic physical quantities.
-# This type may be used to control sound notification emitters.
+#
+# This type may be used to control sound notification emitters assuming the best effort policy:
+# if the requested parameters exceed the capabilities of the emitter, the closest possible values should be assumed.
 
 uavcan.si.unit.frequency.Scalar.1.0 frequency
-# If the requested frequency is outside of the supported range, the command should be ignored.
```

Comment:
Why are we removing this?

Reply:
Because this is a case of overspecification: the generic type intended for modeling sound does not need to define service contracts for sound emitters. If necessary, this will need to be defined externally.